app/controllers/api/users_controller.rb

```diff
@@ -1,17 +1,31 @@
 class API::UsersController < API::RestfulController
```

Comment:
Do you think this might actually be the "CurrentUser" controller? I see Users controller coming into play when we want to show user profiles of other users